pfdp/css/variables.css

```diff
@@ -16,3 +18,27 @@
   --shadow: 0px 10px 10px rgba(0, 0, 0, 0.1);
   --primary: var(--burgundy);
 }
+
+.burgundy {
+  background-color: var(--burgundy);
+}
+
+.red {
```

Comment:
`bg-red`

Reply:
and so on for each color

background color class : `bg-nameofthecolor`
text color class : `nameofthecolor`

Reply:
moved to utilities with the rest of bg color classes